gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -216,17 +292,25 @@ func gcsRequest(w http.ResponseWriter, r *http.Request) {
 	if markers, found := r.URL.Query()["marker"]; found {
 		url += "&marker=" + markers[0]
 	}
+	urlLogger := logger.WithField("url", url)
+
+	// Create a new request using http
+	req, err := http.NewRequest("GET", url, nil)
+
+	if s.tokenGenerator != nil {
+		req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", string(s.tokenGenerator())))
+	}
 
-	resp, err := http.Get(url)
+	resp, err := s.httpClient.Do(req)
 	if err != nil {
-		logger.Printf("GET %s: %s", url, err)
+		urlLogger.WithError(err).Error("error occurred")
 		w.WriteHeader(http.StatusInternalServerError)
 		fmt.Fprintf(w, "http.Get: %v", err)
 		return
 	}
 	defer resp.Body.Close()
 
-	logger.Printf("GET %s: %s", url, resp.Status)
+	urlLogger.WithField("status", resp.Status).Info("URL processed")
```

Comment:
Don't change these error messages?

Reply:
Well, I didn't change the message because all the information from the old message is now in the logger's field.